spec/features/testing_redirect_feed_spec.rb

```diff
@@ -0,0 +1,6 @@
+feature 'testing redirect' do
+  scenario 'it must redirect from / to /chitter_feed' do
+    visit '/'
+    expect(page).to have_content("We have redirected to the Feed")
```

Comment:
Or you could make an expectation about the current url

Reply:
Ok yeah that would work much better in terms of what is displayed when that page loads. Thanks David 😃